Casks/coolterm.rb

```diff
@@ -1,11 +1,16 @@
 cask "coolterm" do
-  version "1.9.0"
+  version "1.9.0.3.1.948"
   sha256 :no_check
 
   url "https://freeware.the-meiers.org/CoolTermMac.zip"
   name "CoolTerm"
   desc "Serial port terminal"
   homepage "https://freeware.the-meiers.org/"
 
+  livecheck do
+    url "http://freeware.the-meiers.org/version/CoolTerm.ver"
```

Comment:
```suggestion
    url "https://freeware.the-meiers.org/version/CoolTerm.ver"
```

I overlooked this when I was commenting on the regex but let's use the HTTPS URL here, since the upstream server supports it.